File: src/main/java/com/viet/news/service/ChannelService.java
package com.viet.news.service;

import com.viet.news.entity.ChannelList;
import com.viet.news.repository.ChannelRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public class ChannelService {

    @Autowired
    public ChannelRepository channelRepository;

    public List<ChannelList> allList() {
        return channelRepository.findAll();
    }
}

File: src/main/java/com/viet/news/service/ChannelService.kt
package com.viet.news.service

import com.viet.news.entity.ChannelList
import com.viet.news.repository.ChannelRepository
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class ChannelService {

    @Autowired
    var channelRepository: ChannelRepository? = null

    fun allList(): List<ChannelList> {
        return channelRepository!!.findAll()
    }
}
